docs/consent/README.md

```diff
@@ -0,0 +1,38 @@
+<h1 align="center"><img src="https://make.hmn.md/altis/Altis-logo.svg" width="89" alt="Altis" /> Consent</h1>
+
+<p align="center">Enhanced user privacy and consent controls for <strong><a href="https://altis-dxp.com/">Altis</a></strong>.</p>
+
+<p align="center"><a href="https://packagist.org/packages/altis/consent"><img alt="Packagist Version" src="https://img.shields.io/packagist/v/altis/consent.svg"></a></p>
+
+## What is the Altis Consent module?
```

Comment:
This title is maybe superfluous now as the following paragraph can act as an introduction following the suggested title above.

Also I think it's important to note the distinction here - Altis Privacy is the module, a module being the high level provider for a set of related features.

Altis Consent API is a feature provided by a plugin that we include in the module.